globus-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# extra requirements for globus
+-r ./requirements.txt
+requests>=2.0.0,<3.0.0
+six==1.10.0
```

Comment:
Weighing in as one of the Globus SDK maintainers.
I can't remember why we pinned the exact version of `six`. We can probably be more permissive. I'll open an issue to dig into it when I have spare cycles.